c#/Makefile

```diff
@@ -0,0 +1,10 @@
+all:
```

Comment:
looks good to me. question. why is the library *.so.1 instead of just *.so? I was confused about this